src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/caller/ReCapSegCaller.java

```diff
@@ -0,0 +1,107 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.caller;
```

Comment:
I'm not touching this class on the assumption that it's not worth our time.

Reply:
No problem.  I did some slight refactoring, but it's otherwise identical to the old code.